### PR TITLE
Liquid swap tool

### LINKED ISSUE #103: Add Liquid Swap tool
Nice, https://github.com/Blockstream/liquid-swap actually supports command line only.

### PR DESCRIPTION
**Pull request by jonasnick**
Fixes #103. ~~Blocked at least until https://github.com/Blockstream/liquid-swap/issues/2 is resolved.~~

### CODE REVIEW

File: pkgs/liquid-swap/default.nix [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+  # We're only interested in the cli and not the gui but we need to add pyqt5
+  # anyway as a build dependency because liquid-swap's setup.py demands it. See
+  # issue https://github.com/Blockstream/liquid-swap/issues/1.
+  checkInputs = [ pyqt5 ];
+  propagatedBuildInputs = [ click ];
+
+  src = fetchurl {
+    urls = [ "https://github.com/Blockstream/liquid-swap/archive/release_${version}.tar.gz" ];
+    sha256 = "c90ba78105469273fb799f412caa754d18bfa310984fa11ffd0091f146cca2ba";
```
Comment (nixbitcoin) [inline pkgs/liquid-swap/default.nix]: Verified pkgs/liquid-swap/default.nix sha256 against `git archive --format=tar --prefix=liquid-swap-release_0.0.1/ release_0.0.1 | gzip -n > release_0.0.1.tar.gz` of gpg signed git tag
* Can't verify 0x09B20724C4DE949B key through WoT

**Comment by nixbitcoin** [comment]
Tested with `nixops deploy` & `liquidswap-cli propose 0 0 0 0`. Can confirm I'm getting the same error https://github.com/Blockstream/liquid-swap/issues/2.

I'll add usage instructions to usage.md once the issue is resolved and I can play around with liquidswap-cli.

**Comment by jonasnick** [comment]
As per https://github.com/Blockstream/liquid-swap/issues/2 wallet needs to be wiped or upgraded. The former made the tool work for me. Removing WIP from the title and ready for review.

**Comment by nixbitcoin** [comment]
Please rebase

File: pkgs/liquid-swap/default.nix [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{ lib, buildPythonPackage, fetchurl, pyqt5, click }:
+
+buildPythonPackage rec {
+  pname = "liquid-swap";
+  version = "0.0.1";
+
+  # We're only interested in the cli and not the gui but we need to add pyqt5
+  # anyway as a build dependency because liquid-swap's setup.py demands it. See
+  # issue https://github.com/Blockstream/liquid-swap/issues/1.
+  checkInputs = [ pyqt5 ];
```
Comment (nixbitcoin) [inline pkgs/liquid-swap/default.nix]: nativeBuildInputs seems more appropriate here

From the nixpkgs manual:
>  nativeBuildInputs ? []: Build-time only dependencies. Typically executables as well as the items listed in setup_requires. 
> checkInputs ? []: Dependencies needed for running the checkPhase. These are added to nativeBuildInputs when doCheck = true. Items listed in tests_require go here.

Comment (jonasnick) [reply pkgs/liquid-swap/default.nix]: fixed

**Comment by jonasnick** [comment]
rebased

**Pull request merged**